Dockerfile.in

```diff
@@ -1,15 +1,15 @@
 FROM UPSTREAM_REPO
 
+MAINTAINER Randy Fay <rfay@drud.com>
```

Comment:
Doesn't matter much to me; I think I added this because the vault one in the vault-consul-on-kube PR had rfay@newmedia in it. I'll leave this alone for now, but am fine with absolutely any policy we come up with.